app/controllers/submissions_controller.rb

```diff
@@ -1,11 +1,16 @@
 class SubmissionsController < ApplicationController
   def new
     @submission = Submission.new
+    @submission.submission_packets.build.build_poem
   end
 
   def create
-    @submission = Submission.create(submission_params_with_user)
+    @submission = current_user.submissions.build(submission_params)
+    @submission.submission_packets.map do |packet|
+      packet.poem.user_id = current_user.id
```

Comment:
I could be wrong here, but it seems like if you have build the `submission` from `current_user` you shouldn't also need to assign it here.